hiring/remote_test/marking_guide.md

```diff
@@ -0,0 +1,32 @@
+# Marking guide
+
+There are no half marks. 70 marks in total. 80% (56 marks) required to pass.
```

Comment:
* Do we think 'pass' or 'fail', is this the right type of message? 
*I also think it would be useful to have some way to judge people based on the quality of the code they submit. There have been plenty of candidates recently who had strong tech chops, who wouldn't be able to complete this easily (primarily as not Rails devs).